script/Arcueid/Day 3/03_00_ARC03_5_1.txt

```diff
@@ -161,22 +173,23 @@ It hurts.
 [sha:cc9abf071161414d4635cb0cef2c90a0b19db9ee]{
 -- Page 256, Offset 4690.
 -- 内臓が焼けているみたいに、痛い。
-It stings, as though my insides are on fire.
+It hurts so bad, it's like my insides are on fire.
 }
 [sha:9a194bb8f111ebe8ea1fbd948f00dd96d9149cf8]{
 -- Page 256, Offset 4691.
 -- 涙は止まらなくて、体はゴミのように地面に崩れ落ちた。
+// Eqv. Offset 5137
 I can't stop crying, and my body crumples to the ground like a piece of discarded garbage.
```

Comment:
You know, I kept this only because 'discarded garbage' has become kind of dear to me.